test/backup.py

```diff
@@ -39,20 +46,21 @@ def setUpModule():
       fd.write(init_db)
       fd.write('''
 # Set real passwords for all users.
-UPDATE mysql.user SET Password = PASSWORD('RootPass')
+UPDATE mysql.user SET %s = PASSWORD('RootPass')
   WHERE User = 'root' AND Host = 'localhost';
-UPDATE mysql.user SET Password = PASSWORD('VtDbaPass')
+UPDATE mysql.user SET %s = PASSWORD('VtDbaPass')
   WHERE User = 'vt_dba' AND Host = 'localhost';
-UPDATE mysql.user SET Password = PASSWORD('VtAppPass')
+UPDATE mysql.user SET %s = PASSWORD('VtAppPass')
   WHERE User = 'vt_app' AND Host = 'localhost';
-UPDATE mysql.user SET Password = PASSWORD('VtAllprivsPass')
+UPDATE mysql.user SET %s = PASSWORD('VtAllprivsPass')
   WHERE User = 'vt_allprivs' AND Host = 'localhost';
-UPDATE mysql.user SET Password = PASSWORD('VtReplPass')
-  WHERE User = 'vt_repl' AND Host = '%';
-UPDATE mysql.user SET Password = PASSWORD('VtFilteredPass')
+UPDATE mysql.user SET %s = PASSWORD('VtReplPass')
+  WHERE User = 'vt_repl' AND Host = '%%';
+UPDATE mysql.user SET %s = PASSWORD('VtFilteredPass')
   WHERE User = 'vt_filtered' AND Host = 'localhost';
 FLUSH PRIVILEGES;
-''')
+''' % (password_col, password_col, password_col, password_col, password_col,
```

Comment:
You can also do the following if you find it more readable:
% tuple([password_col] * 6)